src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,30 @@
+@mixin watch ($watch-size, $border-color, $background-color-watch) {
```

Comment:
mixin is not about separating styles, it is about reusing styles. 
So this watch mixin is not reusable or flexible
I would name it like `circle`, with only   
```
width: $size;
height: $size;
border-radius: 50%;
```
and maybe ` background-color: $background-color-watch;` but with a default value
Other styles i would set directly

Reply:
Ok. I will try to change the code. Thank you!